kickstart/provision.erb

```diff
@@ -143,6 +144,11 @@ echo "updating system time"
 # update all the base packages from the updates repository
 yum -t -y -e 0 update
 
+<% if chef_enabled %>
+echo "Bootstraping chef"
+<%= respond_to?(:chef_bootstrap) ? chef_bootstrap(@host) : snippet_if_exists("chef-client omnibus bootstrap") %>
```

Comment:
if you're interested, here's the foreman_chef PR that adds a snippet(s) https://github.com/theforeman/foreman_chef/pull/11